booster-chat-app/README.md

```diff
@@ -0,0 +1,29 @@
+# Simply chat server powered by Booster Framework
+
+This project is a sample implementation of a chat server in [Booster](https://booster.cloud) to serve as a guidance of the main concepts and artifacts used in the framework.
+
+## Chat requirements
+
+We are going to structure our chat app in chat rooms, which are going to be created or modified only by admins. Inside each chat room the user sends messages to chat with other users.
```

Comment:
By looking at the code, I understand that chat rooms can be created by both, users and admins, and a chat room can be modified only by its owner

Reply:
You're right, they are created by anyone but modified by the admin. Nice catch 👏